Model/Cache.php

```diff
@@ -32,6 +32,11 @@ public function get($key)
     public function store($key, $data)
     {
         global $wpdb;
-        $wpdb->insert($wpdb->prefix.'mobbex_cache', ['cache_key' => $key, 'data' => $data]);
+    
+        // Check that the key is not repeated before inserting the data in the table 
+        $result = $wpdb->get_results("SELECT * FROM " . $wpdb->prefix."mobbex_cache WHERE `cache_key`='$key';");
```

Comment:
Esto no sería lo mismo que hacer `$this->get($key)`?

Reply:
Por lo que tengo entendido en get se elimina la data en cache en un intervalo de 5 minutos. En este caso el error parecía a producirse cuando se guarda prácticamente de forma simultánea la misma data con el mismo id. El error se pudo replicar utilizando la herramienta/script de envio de webhooks que simuló una situación de simultaneidad y es por eso que se hace un checkeo en store justo antes del insert, porque no estaria siendo capturado por get dado el intervalo seteado y, por lo tanto, no lo llega a eliminar produciéndose el error.
Se me ocurre que quizas se podría modificar ese intervalo o hacer la comprobación en get para evitar que se tenga que hacer en store. Fui por implementar el cambio en store porque el error estaba en el guardado.
Respecto a la opción de stack, lo veo como una alternativa viable según sea más práctico evitar que se vuelva a hacer un insert en db para guardar la misma key o reemplazarla en estos casos de simultaneidad.

Reply:
Con replace quedaría en una sóla query, pero estoy de acuerdo en que get sólo debería hacer un `SELECT` a db